mirage/scenarios/collections.ts

```diff
@@ -94,37 +70,135 @@ export function collectionModerationScenario(server: Server, currentUser: ModelI
+                CollectionSubmissionActionTrigger.AdminRemove,
+                'Thanks for being part of our project',
+            ),
+        }  as ProjectBuilderArgument, CollectionSubmissionReviewStates.Removed);
+    });
 
-    rejectedProject({
-        server,
-        currentUser,
-        collection: primaryCollection,
-        license: licensesAcceptable[0],
-        title: 'Rejected Project',
-    } as ProjectModel);
+    /**
+     * Removed By Collection Moderator
+     */
+    [1,2,3].forEach((suffix: number) => {
+        projectBuilder({
+            server,
+            currentUser,
+            collection: primaryCollection,
+            license: licensesAcceptable[0],
+            title: `Removed Project - ${suffix}`,
+            collectionSubmissionActionArgument: buildInitialCollectionSubmissionActionArguments(
+                CollectionSubmissionActionTrigger.ModeratorRemove,
+                'Thanks for being part of our project',
+            ),
+        }  as ProjectBuilderArgument, CollectionSubmissionReviewStates.Removed);
+    });
+
+    /**
+     * Rejected
+     */
+    [1,2,3,4,5,6,7,8,9,10].forEach((suffix: number) => {
+        projectBuilder({
+            server,
+            currentUser,
+            collection: primaryCollection,
+            license: licensesAcceptable[0],
+            title: `RejectedProject - ${suffix}`,
+            collectionSubmissionActionArgument: buildInitialCollectionSubmissionActionArguments(
+                CollectionSubmissionActionTrigger.Reject,
+            ),
+        }  as ProjectBuilderArgument, CollectionSubmissionReviewStates.Rejected);
+    });
+
+    /**
+     * Resubmit after rejected
+     */
+    [1,2,3,4].forEach((suffix: number) => {
+        projectBuilder({
+            server,
+            currentUser,
+            collection: primaryCollection,
+            license: licensesAcceptable[0],
+            title: `Removed Project - ${suffix}`,
+            collectionSubmissionActionArgument: buildInitialCollectionSubmissionActionArguments(
+                CollectionSubmissionActionTrigger.Resubmit,
+                'You are gone ... sorry',
+            ),
+        }  as ProjectBuilderArgument, CollectionSubmissionReviewStates.Rejected);
+    });
+
+    /**
+     * Resubmit after removed
+     */
+    [1,2,3,4,5,6,7,8].forEach((suffix: number) => {
+        projectBuilder({
+            server,
+            currentUser,
+            collection: primaryCollection,
+            license: licensesAcceptable[0],
+            title: `Removed Project - ${suffix}`,
+            collectionSubmissionActionArgument: buildInitialCollectionSubmissionActionArguments(
+                CollectionSubmissionActionTrigger.Resubmit,
+                'I hope to never lose you',
+                suffix % 2 === 1,
+            ),
+        }  as ProjectBuilderArgument, CollectionSubmissionReviewStates.Removed);
+    });
+
+
+    [1,2,3,4,5].forEach((suffix: number) => {
```

Comment:
Since this one creates a bunch of actions with each, this would be the most likely culprit for future slowdowns as mentioned above.

Reply:
Agree, I think we should be ok with just 1 chaos project being the default. If we need more, we can change it locally